packages/bot-web-ui/src/stores/flyout-help-store.js

```diff
@@ -85,7 +89,7 @@ export default class FlyoutHelpStore {
             }
 
             try {
-                await import(/* webpackChunkName: `[request]` */ `../scratch/help-content/help-strings/${block_type}`);
+                await import(/* webpackChunkName: `[request]` */ '@deriv/bot-engine');
```

Comment:
This won't work (as per the other PR), please update.

Reply:
@aaron-binary it works.
`            this.help_string       = help_string_obj.default[block_type];
`